### PR TITLE
Mp create department

### PR DESCRIPTION
**Pull request by thatmikeparrish**
## Link to Ticket
Closes # [Link](https://github.com/NSS-Wacky-Eels/WorkforceManagement/issues/1)

## Description of Proposed Changes
- Users can add departments
- Added ReadMe to project
- Added PR Template

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <MP-CreateDepartment>
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* DepartmentController, Create.cshtml, Delete.cshtml, Edit.cshtml

## Mentions @username

@KaylaReid
@alexfont321
@thatmikeparrish
@taylorlgulley

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Review by taylorlgulley — APPROVED** [review]
Looks good 👍

**Review by KaylaReid — APPROVED** [review]
Looks great!

File: BangazonWorkforce/BangazonWorkforce/Controllers/DepartmentController.cs [hunk 1]
```diff
@@ -130,35 +130,48 @@ public async Task<IActionResult> Edit(int id, Department department)
         // GET: Department/Delete/5
         public async Task<IActionResult> Delete(int? id)
         {
+            string sql = $@"
```
Comment (askingalot) [inline BangazonWorkforce/BangazonWorkforce/Controllers/DepartmentController.cs:133]: Are you using the `sql` variable?

**Review by askingalot — CHANGES_REQUESTED** [review]

File: BangazonWorkforce/BangazonWorkforce/Controllers/DepartmentController.cs (hunk 1 above)
Comment (thatmikeparrish) [reply BangazonWorkforce/BangazonWorkforce/Controllers/DepartmentController.cs:133]: I am not, I can get rid of it.

**Pull request merged**